armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -402,7 +404,7 @@ def _performGeometryTransformations(self, makePlots=False):
                 + "This is a programming error and requires further investigation."
             )
         neutronicsReactor = self.r
-        if self.options.detailedAxialExpansion:
+        if self.options.detailedAxialExpansion or self.options.nonUniformAssems:
             converter = self.geomConverters.get("axial")
```

Comment:
To make this more efficient, I recommend that we have an option to handle remeshing _only_ the nonUniformAssems if possible. Remeshing the entire reactor will be time consuming and actually not needed here. I will look at the code and suggest a change.

Reply:
Yeah, I tried an approach like this but what I had ended up being just as time consuming as remeshing the whole reactor. I can show you what I had and get your thoughts.

Reply:
Okay I'm good then